spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/actions/TestDeleteReachableFilesAction.java

```diff
@@ -94,6 +96,22 @@ public class TestDeleteReachableFilesAction extends SparkTestBase {
           .withPartition(TestHelpers.Row.of(3))
           .withRecordCount(1)
           .build();
+  static final DeleteFile FILE_A_POS_DELETES =
+      FileMetadata.deleteFileBuilder(SPEC)
+          .ofPositionDeletes()
+          .withPath("/path/to/data-a-pos-deletes.parquet")
+          .withFileSizeInBytes(10)
+          .withPartitionPath("c1=0") // easy way to set partition data for now
```

Comment:
Small nit: can it be withPartition like other ones?

Reply:
Yep, changed that.